spec/tapioca/rbi/model_spec.rb

```diff
@@ -97,6 +100,9 @@ class ModelSpec < Minitest::HooksSpec
           cls << singleton_class
           assert_equal("::Foo::Bar::<self>", singleton_class.to_s)
 
+          struct = RBI::Struct.new("Foo", members: ["foo", "bar"], keyword_init: true)
```

Comment:
We might want to add special treatment for it in [Index](https://github.com/Shopify/tapioca/blob/master/lib/tapioca/rbi/index.rb#L43) so we can also index the members as if they were real methods. So we can detect conflicting redefinitions / duplicates.